--- /etc/nginx/sites-enabled/components-aggregator.conf.backup       2025-09-20 11:32:22.693449420 +0300
+++ /etc/nginx/sites-enabled/components-aggregator.conf      2025-09-20 11:33:06.135843982 +0300
@@ -3,7 +3,10 @@
   server_name _;
 
   # health
-  location /_health { return 200 'ok'; add_header Content-Type text/plain; }             
+  location /_health { return 200 'ok'; add_header Content-Type text/plain; }             
+
+  # статические файлы
+  include snippets/loads-static.conf;       
 
   # основной сервер (Node на 3000)
   location / {